src/domain/event/service/event.service.ts

```diff
@@ -1,23 +1,62 @@
 import {
-  getEventById,
   getEvents,
+  getEventById,
+  saveEvent,
+  updateEvent,
+  deleteEvent,
 } from '@domain/event/repository/event.repository';
-import { ErrorSubCategoryEnum } from '@root/src/common/exception/enum';
-import { CallerWrongUsageException } from '@root/src/common/exception/internal.exception';
+import { CallerWrongUsageException } from '@common/exception/internal.exception';
+import { ErrorSubCategoryEnum } from '@common/exception/enum';
 
 export async function getAkcseEvents() {
   return await getEvents();
 }
 
 export async function getAkcseEventById(id: number) {
-  const record = await getEventById(id);
+  const event = await getEventById(id);
 
-  if (!record) {
+  if (!event) {
     throw new CallerWrongUsageException(
       ErrorSubCategoryEnum.INVALID_INPUT,
       'no such event',
     );
   }
 
-  return record;
+  return event;
+}
+
+export async function saveAkcseEvent(param: {
+  title: string;
+  description: string;
+  fee: number;
+  startDateTime: Date;
+  endDateTime: Date;
+  location: string;
+  signUpDeadline: Date;
+  rsvpLink?: string;
+  imageUrl?: string;
+  updatedAt: Date;
+}) {
+  return await saveEvent(param);
```

Comment:
```suggestion
await saveEvent(param);
```
As discussed in this [PR](https://github.com/AKCSE-MB/AKCSE-Server/pull/39), we are not using the return value of `saveEvent` (`updateEvent`, and `deleteEvent`). Could you remove `return`?